dashboard/app/views/devise/registrations/_finish_sign_up.html.haml

```diff
@@ -97,21 +97,12 @@
       .span5
         = f.select :age, age_options, include_blank: true
 
-    - if ask_gender_experiment?(request, session.id.to_s)
-      - gender_input_type = gender_input_type?(request, session)
-      - SignUpTracking.log_gender_input_type_started(session, gender_input_type, request.locale, 'email_signup')
-      - if gender_input_type != User::GenderExperimentHelper::GENDER_INPUT_TYPE_NONE
-        .row#gender-dropdown{style: "display: none;"}
-          .span3.signup-field-label
-            = f.label :gender, t('signup_form.gender')
-          - if @user.errors[:gender].present?
-            %span.error= @user.errors[:gender].join(', ')
-          - if gender_input_type == User::GenderExperimentHelper::GENDER_INPUT_TYPE_DROPDOWN
-            .span5
-              = f.select :gender, gender_options
-          - else
-            .span5
-              = f.text_field :gender
+    - if experiment_value('gender')
+      .row#gender-field
+        .span3.signup-field-label
+          = f.label :gender, t('signup_form.gender')
+        .span5
+          = f.text_field :gender_student_input, maxlength: 50
```

Comment:
```suggestion
        .span3.signup-field-label
          = f.label :gender_student_input, t('signup_form.gender')
        .span5
          = f.text_field :gender_student_input, maxlength: 50
```
The label and field names should match. This should produce HTML such that the <label> is associated to the <input> using the `for` attribute which is important for a11y tools. Double check the HTML this produces does this.